pilot/pkg/xds/ads.go

```diff
@@ -389,8 +381,8 @@ func (s *DiscoveryServer) handleEds(con *Connection, discReq *discovery.Discover
 	if discReq.ResourceNames == nil && discReq.ResponseNonce != "" {
 		return nil
 	}
-	con.Clusters = discReq.ResourceNames
-	adsLog.Debugf("ADS:EDS: REQ %s clusters:%d", con.ConID, len(con.Clusters))
+	con.node.Active[v3.EndpointShortType].ResourceNames = discReq.ResourceNames
```

Comment:
very Nit: may be worth a comment on WatchedResources that ResourceNames for EDS is clusters, and ResourceNames for CDS is empty. It makes sense but may not be immediately obvious

Reply:
Sure. Will add comment in a follow-up.